From 365c576c35073bff9f56f32ee1e68a0897ec1d6f Mon Sep 17 00:00:00 2001
From: Nicolas Le Bayon <nicolas.le.bayon@st.com>
Date: Tue, 24 Oct 2023 11:36:10 +0200
Subject: [PATCH 0592/1141] arm64: dts: st: add cma alloc-range on
 stm32mp257f-ev1

This new property defines the address range in which the cma region can
be allocated. It must be located in the 32-bit physical address area.

Signed-off-by: Nicolas Le Bayon <nicolas.le.bayon@st.com>
Change-Id: Ibc02dd43249dd33cdefec50b7677977e0ee60fd0
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/335805
Reviewed-by: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Domain-Review: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
---
 arch/arm64/boot/dts/st/stm32mp257f-ev1-ca35tdcid-resmem.dtsi | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm64/boot/dts/st/stm32mp257f-ev1-ca35tdcid-resmem.dtsi
+++ b/arch/arm64/boot/dts/st/stm32mp257f-ev1-ca35tdcid-resmem.dtsi
@@ -168,6 +168,7 @@
 		linux,cma {
 			compatible = "shared-dma-pool";
 			reusable;
+			alloc-ranges = <0 0x80000000 0 0x80000000>;
 			size = <0x0 0x8000000>;
 			alignment = <0x0 0x2000>;
 			linux,cma-default;
